api/MediaDevices.json

```diff
@@ -365,40 +365,14 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
-            "chrome": [
-              {
-                "version_added": "52"
-              },
-              {
-                "version_added": "47",
-                "version_removed": "52",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Experimental Web Platform features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "52"
-              },
-              {
-                "version_added": "47",
-                "version_removed": "52",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Experimental Web Platform features",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."
-              }
-            ],
+            "chrome": {
+              "version_added": "53",
+              "notes": "Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."
```

Comment:
This is a tough note to get right. If this note is going to remain, then we should resolve some ambiguity of "Older versions of Chrome" and the relationship of this method to the older, prefixed one.

We could either reflect the data in the other method:

> From Chrome 21, Chrome implements a similar method, <code>navigator.webkitGetUserMedia()</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia()</code></a> method.

Or we could be a bit more circumspect (and insulate ourselves from the other method's details):

> Some versions of Chrome before 53 may implement a similar method, <code>navigator.webkitGetUserMedia()</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia()</code></a> method.

Reply:
Oh my, when trying decide here I noticed that the data for when the prefixed variant was introduced also seems to be wrong. Will need to sort that out too...

Reply:
Can I do anything to help this along, @foolip?

Reply:
No, I've just left it sitting, it's not blocked on anything. It's bumped up my list now, I'll finish it in the coming days.

Reply:
OK, so I've used http://mdn-bcd-collector.appspot.com/tests/api/Navigator/webkitGetUserMedia to try and pin this down, but now find that that on both macOS and Windows, `navigator. webkitGetUserMedia` _was_ introduced in Chrome 21. I'm not sure what it was I tested 3 weeks ago.

The data for `api.Navigator.getUserMedia` looks all correct already, and going over the changes I've made here, I don't think anything is missing. It's ready for review as-is.

Reply:
As for the wording of the note, since I've now confirmed this was added in Chrome 21, the "From Chrome 21, Chrome implements a similar method" wording would work. However, note that we also have the same wording for Firefox, "Older versions of Firefox ..."

We also have "Older versions of" notes for `api.EventTarget.dispatchEvent` and other `css.properties.unicode-bidi.isolate` Is the problem that a reader wouldn't be able to figure out which versions this refers to? I don't think I see the problem myself, so I can't have a preference on how to fix it.

Reply:
The way I see it, "Older versions" could be interpreted as relative to `"version_added"` (in this case, the range of versions 1-52) or relative to the current version of the browser (the range 1-84). I suppose with the note, we're trying to say:

```
If you need this capability before version 53, refer to <code>navigator.webkitGetUserMedia</code>, a prefixed form of the deprecated <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API.
```

Maybe that's more helpful and easier to apply to the Firefox notes?

Reply:
Thanks @ddbeck! Sorry it took me a while, but I've updated to use your notes, and that worked out well I think.